examples/hotrod/services/config/config.go

```diff
@@ -33,18 +34,19 @@ var (
 	// MySQLGetDelayStdDev is standard deviation
 	MySQLGetDelayStdDev = MySQLGetDelay / 10
 
-	// RouteWorkerPoolSize is the size of the worker pool used to query `route` service
-	RouteWorkerPoolSize = 3
+	// MySQLMutexDisabled controls whether there is a mutex guarding db query execution.
+	// When not disabled it simulates a misconfigured connection pool of size 1.
+	MySQLMutexDisabled = false
```

Comment:
Double negatives are confusing :-) How about `MySQLMutexEnabled = true` ?

Reply:
This isn't double negative, but more importantly disabled plays nicer with false as the default value.